### PR TITLE
fix: Update paths for webmanifest and add svg.

### PR DESCRIPTION
**Pull request by froboy**
https://yusa.atlassian.net/browse/DS-1287

Checker failed a couple places. Cleaning those up. I think that’s as good as we’re going to be able to do.

- the hardcoded paths won’t work for the builds as they’re in subdomains, but they will work in most other sites.
- browserconfig.xml isn’t detected at the root path, but I’ve added an entry which should get rewritten with the configured path so it’s found.
- I added an SVG favicon which is now supported as well.

## Steps for review

- [ ] Check at https://realfavicongenerator.net/favicon_checker

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] Try to use Conventional commit messages accordingly to the standard https://www.conventionalcommits.org/en/v1.0.0/#specification
- [ ] [Documentation](https://github.com/ycloudyusa/yusaopeny/tree/main/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ycloudyusa/yusaopeny/blob/main/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with an account on drupal.org, otherwise, you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ycloudyusa/yusaopeny/blob/main/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by froboy** [comment]
ok to test

**Comment by froboy** [comment]
retest please

**Comment by froboy** [comment]
retest this please

**Comment by froboy** [comment]
retest this please

File: openy.install [hunk 1]
```diff
@@ -1311,3 +1311,18 @@ function openy_update_8097() {
...
+function openy_update_8098() {
+  $path = \Drupal::service('extension.list.profile')->getPath('openy') . '/config/optional';
+  /** @var \Drupal\config_import\ConfigImporterService $config_importer */
+  $config_importer = \Drupal::service('config_import.importer');
+  $config_importer->setDirectory($path);
+  $config_importer->importConfigs([
+    'responsive_favicons.settings',
+  ]);
+  // Clear all cache to rebuild cached favicon
+  drupal_flush_all_caches();
```
Comment (podarok) [inline openy.install:1327]: is there a way to invalidate only needed cache bin without flushing all caches?
upgrade path is heavy (

**Review by podarok — APPROVED** [review]

File: openy.install (hunk 1 above)
Comment (podarok) [reply openy.install:1327]: @froboy

**Pull request merged**